source/parser/util.civet

```diff
@@ -229,9 +229,18 @@ function insertTrimmingSpace(target: ASTNode, c: string): ASTNode
       ...target
       token: target.token.replace(/^ ?/, c)
     }
+  else if target <? "string"
+    target.replace(/^ ?/, c)
   else
     target
 
+/**
+ * Trims the first single space from the spacing array or node's children if present
+ * maintains $loc for source maps
+ */
+function trimFirstSpace(target: ASTNode): ASTNode
+  insertTrimmingSpace target, ""
```

Comment:
We use this a zillion times. I wonder if just giving `insertTrimmingSpace` an optional second argument (defaulting to empty string) would be better? Possibly also renaming it to `trimFirstSpace`...

Reply:
Maybe `trimFirstSpaceAndInsert` and keep the `trimFirstSpace` alias.

Reply:
`replaceFirstSpace`?